news/2021-03-02-mappers-report-february.md

```diff
@@ -0,0 +1,114 @@
+
+Do you want to see your own name on this list? Do you like modding and helping other people with their beatmaps? If your answer to both of these questions is "yes", then you can also become a Beatmap Nominator! The applications are always open.
+
+Head over to the [Beatmap Nominator website](https://bn.mappersguild.com "Beatmap Nominators") and apply today!
+
+### BN Resignations
+
+- **osu!:** [Nao Tomori](https://osu.ppy.sh/users/5364763)
+- **osu!catch:** [Rocma](https://osu.ppy.sh/users/566276)
+
+### NAT Retirement
+
+After exactly 3 years as QAT/NAT, [Lasse](https://osu.ppy.sh/users/896613) decided to step down from the NAT. But he won't be completely gone as he will continue to be present within the mapping and modding community as Beatmap Nominator now, raising his modding activity even higher.
+
+Thank you for all the hard work!
+
+## <a id="mappers-showcase"></a>Mappers' Showcase
+
+Welcome to the 3rd cycle of the Mappers' Showcase! Twice already have we shown you the secret mapper gems in the community and now we start all over again with the osu! game mode!
+
+A lot of thanks to this months' selectors [Morrighan](https://osu.ppy.sh/users/12042090) and [TheKingHenry](https://osu.ppy.sh/users/5128277)!
+
+### Val
+
+Hailing from the newest generation of metal mappers, [Val](https://osu.ppy.sh/users/9010492) is perhaps the most known in the community thus far from his tandem mapping effort with [SinHay](https://osu.ppy.sh/users/8855704) on the monumental mapset [Imperial Circus Dead Decadence - Songs Compilation](https://osu.ppy.sh/beatmapsets/1081431), which landed in the Loved-section at the beginning of this year. While somewhat subdued in the not-so-uncommon style of compilations, this debut with its wide variety of songs at various tempos already shows a good understanding of the fundamentals and more, particularly when it comes to streams.
+
+This prowess can be further seen on his more recent mapset [Ne Obliviscaris - Devour Me, Colossus (Part I): Blackholes](https://osu.ppy.sh/beatmapsets/1274508), where the expressiveness of the pattern design has been noticeably cranked up. In addition to the flowing streams, this marathon map also features beautifully crafted calmer middle sections where Val showcases his ability to work with sliders and rhythmically sparser grooves.
+
+For the fans of this style of Val, [Native Construct - Mute](https://osu.ppy.sh/beatmapsets/1250251) is there to satiate the hunger. Featuring only relatively brief sections of pure streams, the rest of the song - and map - is populated by more diverse sections ranging from funky to serene.
+
+Even if you prefer metal as a type of element rather than music, fret not. A glimpse at the six-feet deep section of Val's mapping career clearly shows that anything is possible in the music department too. Regardless, this is a name you don't want flying under your radar in the future, if fluid streams or beautiful sliderwork are close to your heart.
```

Comment:
uber nitpick
```suggestion
Even if you prefer metal as a type of element rather than music, fret not. A glimpse at the six-feet deep section of Val's mapping career clearly shows that anything is possible in the music department too. Regardless, this is a name you don't want flying under your radar in the future if fluid streams or beautiful sliderwork are close to your heart.
```